packages/devtools_app/lib/src/screens/debugger/debugger_model.dart

```diff
@@ -391,11 +391,13 @@ class FileNode extends TreeNode<FileNode> {
     if (other is! FileNode) return false;
     final FileNode node = other;
 
-    if (scriptRef == null) {
-      return node.scriptRef != null ? false : name == node.name;
-    } else {
-      return node.scriptRef == null ? false : scriptRef == node.scriptRef;
-    }
+    return scriptRef == null
+        ? node.scriptRef != null
+            ? false
+            : name == node.name
+        : node.scriptRef == null
+            ? false
+            : scriptRef == node.scriptRef;
```

Comment:
the nested ternary operators makes this a bit difficult to read

Reply:
agreed this is overly clever. reverted.